templates/program/nph/order/sample-modify.html.twig

```diff
@@ -36,37 +36,50 @@
         </thead>
         <tbody class="modify-samples">
         {% for index, nphSample in order.nphSamples %}
-            <tr class="row-samples">
+            {% if nphSample.modifyType == constant('App\\Entity\\NphSample::CANCEL') and nphSample.sampleCode in activeSamples %}
+                {% set sampleClass = 'no-restore' %}
+            {% else %}
+                {% set sampleClass = '' %}
+            {% endif %}
+            <tr class="row-samples {{ sampleClass }}">
                 <td>
                     {{ form_widget(sampleModifyForm[nphSample.sampleCode]) }}
                 </td>
                 <td>
-                    ({{ index + 1 }}) {{ samples[nphSample.sampleCode] }} ({{ nphSample.sampleId }})
-                    {% if nphSample.modifyType == 'cancel' %}
+                    <p>({{ index + 1 }}) {{ samples[nphSample.sampleCode] }} ({{ nphSample.sampleId }})
+                    {% if nphSample.modifyType == constant('App\\Entity\\NphSample::CANCEL')%}
                         <span class="text-danger">Cancelled</span>
                     {% else %}
                         <span class="text-success">Active</span>
                     {% endif %}
+                    </p>
                 </td>
                 <td>
+                    <p>
                     {% if nphSample.collectedTs is not empty %}
                         <i class="fa fa-check text-success" aria-hidden="true"></i>
                         {{ nphSample.collectedTs|date('n/j/Y g:ia', app.user.timezone) }}
                         {{ nphSample.collectedUser.email }}
                     {% else %}
                         <i class="fa fa-times text-danger"></i>
                     {% endif %}
+                    </p>
                 </td>
                 <td>
+                    <p>
                     {% if nphSample.finalizedTs is not empty %}
                         <i class="fa fa-check text-success" aria-hidden="true"></i>
                         {{ nphSample.finalizedTs|date('n/j/Y g:ia', app.user.timezone) }}
                         {{ nphSample.finalizedUser.email }}
                     {% else %}
                         <i class="fa fa-times text-danger"></i>
                     {% endif %}
+                    </p>
                 </td>
             </tr>
+            {% if nphSample.modifyType == 'cancel' and nphSample.sampleCode in activeSamples %}
```

Comment:
@Ehaic Sorry, I missed this.Could you use class constants here as well?

Reply:
Yep missed that too, I'll double check the whole file this time.